samples/bluetooth/mesh/light_switch/README.rst

```diff
@@ -34,21 +34,29 @@ This sample application has the following composition data:
    Gen. OnOff Client
    =================  =================  =================  =================
 
-The :ref:`bt_mesh_onoff_cli_readme` instances in elements 1-4 are controlled by the buttons on the Device Kit.
-The Config server allows configurator devices to configure the node remotely.
-The Health server is used to call attention to the device during provisioning.
+The models are used for the following purposes:
 
-The model handling is implemented in src/model_handler.c, which uses the :ref:`dk_buttons_and_leds_readme` to detect button presses on the board.
+* :ref:`bt_mesh_onoff_cli_readme` instances in elements 1 to 4 are controlled by the buttons on the development kit.
+* Config Server allows configurator devices to configure the node remotely.
+* Health Server is used to call attention to the device during provisioning.
+
+The model handling is implemented in :file:`src/model_handler.c`, which uses the :ref:`dk_buttons_and_leds_readme` to detect button presses on the board.
 
 The model handler calls :cpp:func:`bt_mesh_onoff_cli_set` to turn the LEDs of a Mesh Light device on or off.
-As we pass a response buffer to the function, it will will block until the model receives a response from the target device.
-If the function returns successfully, the response buffer will contain the response from the Mesh Light.
-In this example, the contents of the response will be ignored, as we handle all incoming OnOff status messages in the status callback.
-Note that the response structure is still required, as the Generic OnOff Client model won't request a response from the Server if we pass ``NULL`` in the response parameter.
+As a response buffer is passed to the function, it blocks until the model receives a response from the target device.
+If the function returns successfully, the response buffer will contain the response from Mesh Light.
```

Comment:
> ... the response from Mesh Light

the response from the Mesh Light device? It's the device that sends the response, not the example, if you get what I mean